configs/opencv-ovms/demos/classification/python/entrypoint.sh

```diff
@@ -12,6 +12,10 @@ then
 	mqttArgs="--mqtt ${MQTT}"
 fi
 
+echo "Waiting for OVMS server to be ready...."
+sleep 10
+echo "Done Waiting...."
+
```

Comment:
please move this logic to the docker-compose specifically as this entrypoint script also used by profile-launcher case